multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/jobs/TokensCleaner.java

```diff
@@ -2,53 +2,43 @@
 
 import static java.text.MessageFormat.format;
 
-import java.util.Collection;
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.time.ZonedDateTime;
 import java.util.Date;
-import java.util.List;
-import java.util.stream.Collectors;
 
 import javax.inject.Inject;
 import javax.inject.Named;
 
-import org.cloudfoundry.multiapps.controller.core.util.SecurityUtil;
+import org.cloudfoundry.multiapps.controller.persistence.services.AccessTokenService;
 import org.cloudfoundry.multiapps.controller.process.Messages;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.core.annotation.Order;
-import org.springframework.security.oauth2.common.OAuth2AccessToken;
-import org.springframework.security.oauth2.provider.token.TokenStore;
 
 @Named
 @Order(10)
 public class TokensCleaner implements Cleaner {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(TokensCleaner.class);
 
-    protected final TokenStore tokenStore;
+    protected final AccessTokenService accessTokenService;
 
     @Inject
-    public TokensCleaner(@Named("tokenStore") TokenStore tokenStore) {
-        this.tokenStore = tokenStore;
+    public TokensCleaner(AccessTokenService accessTokenService) {
+        this.accessTokenService = accessTokenService;
     }
 
     @Override
     public void execute(Date expirationTime) {
-        Collection<OAuth2AccessToken> tokens = tokenStore.findTokensByClientId(SecurityUtil.CLIENT_ID);
+        LocalDateTime date = ZonedDateTime.now()
```

Comment:
Why so long period? We could delete all tokens expired before ZonedDateTime.now()